tinkerbell/stack/values.yaml

```diff
@@ -49,7 +49,12 @@ stack:
     # sourceInterface is the Host/Node interface to use for listening for DHCP broadcast packets.
     # When unset, the interface from the default route will be used.
     # sourceInterface: eno1
-    # TODO(jacobweinstock): add feature to be able to disable listening for broadcast traffic.
+    # enable/disable listening for broadcast traffic. Useful when the environment employs DHCP relay agent(s).
+    # default is true.
+    listenBroadcastTraffic: true
+    # interfaceMode determines how we create the interface needed to listen for DHCP broadcast traffic.
+    # by default macvlan is used. ipvlan is the only other option.
+    # interfaceMode: ipvlan
```

Comment:
[Static defaults should generally reside in the values.yaml](https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function) so consumers don't need to hunt through template code to find it.

> In an actual chart, all static default values should live in the values.yaml, and should not be repeated using the default command (otherwise they would be redundant). However, the default command is perfect for computed values, which cannot be declared inside values.yaml

Reply:
updated